plugin/src/helpers/styleguide.ts

```diff
@@ -1,12 +1,13 @@
 import {packageName} from '../constants'
 import {TranslateStyleguide, TranslateStyleguideContext} from '../translate/types'
 
-export function validateStyleguide(styleguide: string) {
-  if (styleguide.length > 2000) {
+export function validateStyleguide(styleguide: string | undefined) {
+  if (styleguide && styleguide?.length > 2000) {
```

Comment:
this should be sufficient

```suggestion
  if (styleguide?.length > 2000) {
```

or this, if you already check if its falsy
```
if (styleguide && styleguide.length > 2000) {
```

Reply:
You are right that the ? Is redundant tho

Reply:
with optional chaining, this works as expected for me (returning `false`)
```
const test = undefined
console.log(test?.length > 200)
```

Reply:
at runtime yes, but typescript does not accept it 
![image](https://github.com/user-attachments/assets/98b5d0f5-db89-48c8-8a85-15af3a99233d)

Reply:
ah okay, i see. No worries then 👍